.github/workflows/ci.yml

```diff
@@ -41,3 +41,29 @@ jobs:
         # Remember that you can parallelise this by using strategy.matrix
         run: |
           nextflow run ${GITHUB_WORKSPACE} -profile test,docker --outdir ./results
+
```

Comment:
If I add a new step to the first job, will this make a 2D matrix of NXF_VER x config? I would be fine to only run the two version tests with the test profile and then have a separate job for checking the different profiles instead of having each profile tested with each nextflow version

Reply:
Ahh, yes, I did miss something of what was going on there, apologies.

If we were to test additional profiles, I think it would make sense to test them agains the nextflow versions as well, so you would just add the additional profiles to the existing matrix tests and pass them like `-profile${{ matrix.profile}},docker`.

But again, not running the full tests in CI is a conscious decision- they're intended for release time.

Reply:
Ah hm, so you mean run both nextflow versions with every config? Alright, can also do that.

Sure, that is no problem, I'll remove the full_test